### ISSUE TITLE
Build is out of date

### ISSUE THREAD

**Issue by mholl1983** [russinnes/RT-N600-Padavan#2]
Any plans for a new build? Thanks!

**Comment by mholl1983** [comment]
Can you post the steps you did to compile the RT n600 firmware so we can build it ourselves?

**Comment by russinnes** [comment]
OpenWRT 21.02.3 squashfs (sysupgrade) for the Asus RT-N12 VP B1 is working on this box. No 5GHz yet, but I'm still playing with it. If someone wants more functionality and expandability (openwrt) vs padavan. 
Note: I had to use their firmware rescue utility to flash the openwrt binary.

**Comment by mholl1983** [comment]
That's great! Thanks a lot for the update here! I was just thinking about getting rid of the router but will hold onto it now and gladly test out OpenWRT. Will gladly donate to the effort too. Let me know if I can help.

**Comment by russinnes** [comment]
> That's great! Thanks a lot for the update here! I was just thinking about getting rid of the router but will hold onto it now and gladly test out OpenWRT. Will gladly donate to the effort too. Let me know if I can help.

Just see my notes on the readme referring the switch ports. Either adjust the switch or use LAN1 for WAN (default). The LR2100 binary is your best choice if you dont mind no LED's for now. https://downloads.openwrt.org/releases/21.02.2/targets/ramips/mt76x8/openwrt-21.02.2-ramips-mt76x8-totolink_lr1200-squashfs-sysupgrade.bin

**Comment by mholl1983** [comment]
Hi, any pointers on the recovery tool? The OpenWRT documentation suggests to use recovery option onboard the D-Link device. I can't find a specific OpenWRT rescue tool, and the D-Link recovery tool didn't work for me.

**Comment by russinnes** [comment]
Grab the asus recovery tool from their website. Instructions are there, basically set a static ip, power it up holding reset until LED flashes fast, then flash away. You can do this after it’s bricked as well, it will recover. It’s just a glorified Tftp, little less work to use.

**Comment by mholl1983** [comment]
K, that's what I was using but I'll try it again and double-check the steps, thanks! (PS. I'm a tech docs writer. If you need any volunteer documentation as this develops, I'd be happy to help in spare time.)

**Comment by russinnes** [comment]
Asus, not dlink

**Comment by mholl1983** [comment]
Yes, I mistyped above; I meant Asus, sorry.

**Comment by mholl1983** [comment]
Tried again slowly and same result: works for stock firmware restoration, fails for OpenWRT firmware. I've done several device recoveries so comfortable w/ steps, but happy to be told I'm missing something obvious. Above linked firmware is correct?

**Comment by russinnes** [comment]
Yes. As posted, don’t rely on leds. Wait for it to boot, watch for an IP assignment, then open 192.168.1.1 in your browser. It takes about 69 secs or so. The LEDS are not mapped for this board, so they won’t work. I’ll add them with a config for this board properly when I get some time.

**Comment by russinnes** [comment]
I've built a proper image for the N600 with all LED's, 2g/5g wifi, USB (except 5g wireless LED, can't find it's GPIO address yet). I'll upload it here in an hour.

**Comment by russinnes** [comment]
Here you go:
https://github.com/russinnes/ASUS-RT-N600-OpenWRT

**Comment by russinnes** [comment]
I should mention - any port mapping of the switch that you did with my earlier binary -- dont do it now, do as you normally would. The switch mapping now reflects what you see on the back of the device.
